File: core_common/src/main/java/com/huyingbao/core/util/CommonUtils.java
package com.huyingbao.core.util;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by liujunfeng on 2019/1/1.
 */
public class CommonUtils {

    /**
     * 显示短暂的Toast
     *
     * @param context
     * @param text
     */
    public static void showShortToast(Context context, String text) {
        Toast.makeText(context, text, Toast.LENGTH_SHORT).show();
    }
}

File: core_common_kotlin/src/main/java/com/huyingbao/core/common/util/CommonUtils.kt
package com.huyingbao.core.common.util

import android.app.Application
import android.widget.Toast

import javax.inject.Inject
import javax.inject.Singleton

/**
 * Created by liujunfeng on 2018/12/28.
 */
@Singleton
class CommonUtils @Inject
internal constructor(mApplication: Application) {

    private val toast: Toast

    init {
        toast = Toast.makeText(mApplication, "", Toast.LENGTH_SHORT)
    }


    /**
     * 显示短暂的Toast
     *
     * @param text
     */
    fun showShortToast(text: String) {
        toast.cancel()
        toast.setText(text)
        toast.show()
    }
}
